src/main/java/cceclipseplugin/ui/dialogs/DeleteProjectDialog.java

```diff
@@ -68,6 +74,16 @@ protected void createButtonsForButtonBar(Composite parent) {
 	@Override
 	protected void okPressed() {
 		PluginManager.getInstance().getRequestManager().deleteProject(project.getProjectID());
+		// TODO: move out to metadata manager or request manager?
```

Comment:
I'm thinking it would be a good idea to move it to the RequestManager. Less actual functionality in the UI code is probably better.